appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -242,7 +241,9 @@ public class Appcues internal constructor(koinScope: Scope) {
      * @param frame frame used to inflate the embedded experience
      */
     public fun registerEmbed(frameId: String, frame: AppcuesFrameView) {
-        renderContextManager.registerEmbedFrame(frameId, frame)
+        appcuesCoroutineScope.launch {
+            experienceRenderer.startFrame(frameId, frame)
```

Comment:
instead of registering we "start", this means that we will also try to init an experience to this frame if exists